draft-ralston-mimi-terminology.md

```diff
@@ -152,6 +152,24 @@ in the same logical place as a client (i.e.: in peer-to-peer environments), howe
 default assumption if not clarified is that the client and server are two different
 entities.
 
+**Hub Server**: The specific server responsible for routing events to other servers in the
+context of a chat. While MIMI has generally agreed that a hub server is required, some systems
+may not define such a role. Sometimes this is shortened to **Hub**.
+
+**Owning Server**: The specific server responsible for applying access control to a chat. When
+the system requires more than one (or all) servers to apply access control, there is no owning
+server. The owning server will typically be the hub server for a chat. This should NOT be
+shortened to "owner" to avoid confusion with other definitions in this document.
+
+**Server Owner**: The entity or person responsible for administrative function of the server,
+such as creating user accounts, banning/blocking chats, etc. This should NOT be shortened to
+"owner" to avoid confusion with other definitions in this document.
+
+**Server Operator**: The entity or person responsible for uptime and availability of a given
+server. This is typically the same as the server owner, though servers may be deployed to a
+shared hosting environment. The hosting provider becomes the server operator while the account
+holder with that provider becomes the server owner.
```

Comment:
It's fairly unlikely that hosting providers for MIMI implementations/transports will sprout up (it just doesn't make sense), but it feels distinct enough from server owners to be worth writing down.